packages/utilities/psammead-test-helpers/README.md

```diff
@@ -0,0 +1,36 @@
+# psammead-test-helpers &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-test-helpers.svg)](https://www.npmjs.com/package/@bbc/psammead-test-helpers) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+
+This package is a collection helper methods used for snapshot testing, with support for styled components.
+
+## Installation
+
+```jsx
+npm install @bbc/psammead-test-helpers --save
+```
+
+## Usage
+
+```jsx
```

Comment:
I think it should be important to explain what the `shouldMatchSnapshot` function does. e.g. It renders the component passed the second argument using `react-test-renderer`, converts it to JSON and asserts that it matches the given snapshot, which will be saved in the `__snapshots__` directory. 
The first argument is the title for the test. 

Likewise, since we've added `shallowRender`,  `shouldShallowMatchSnapshot`, `isNull`, we should explain these as well.